arlpy/uwa.py

```diff
@@ -167,20 +167,23 @@ def doppler(speed, frequency, c=soundspeed()):
     """
     return (1+speed/float(c))*frequency
 
-def bubble_resonance(radius, depth=0):
-    """Get the resonant frequency of a bubble of a given radius.
-
-    The bubble resonance is computed based on Medwin & Clay (1998).
-
-    :param radius: radius of the bubble in m
-    :param depth: depth in m
-    :returns: resonant frequency of the bubble in Hz
+def bubble_resonance(radius, depth=0.0, gamma = 1.4, p0 = 1.013e5, rho_water = 1022.476):
+    """Compute resonance frequency of a freely oscillating has bubble in water, given:
+- bubble `radius` in meters
```

Comment:
Please use the doc-string format as per the original doc-string and rest of the code base. The format is needed for proper rendering using sphinx.